menu/menu.yaml

```
@@ -118,6 +118,26 @@
       - Title: Observability
         Url: architecture/aws/observability
 
+- Name: Particular Service Platform
+  Topics:
+  - Title: General
+    Articles: 
+    - Url: platform
+      Title: About the Service Platform
+    - Url: platform/installation
+      Title: Platform Installation
+    - Url: platform/platform-sample-package
+      Title: Platform Sample Installation
```

```suggestion
      Title: Platform Sample installation
```
I'm on the fence on whether Sample needs capitalization here but installation definitely doesn't to match the rest of the menu items